presto-product-tests/bin/product-tests-suite-6-non-generic.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+# Suite of product tests which are *not* run on ${HADOOP_BASE_IMAGE} and therefore
+# there is no point in running them several times, for different Hadoop distributions.
+
+exit_code=0
+
+# Uses hadoop, but it irrelevant from these tests' perspective.
+presto-product-tests/bin/run_on_docker.sh \
+    singlenode-ldap \
+    -g ldap \
+    -x simba_jdbc \
+    || exit_code=1
+
+# We have docker images with KMS on CDH only. TODO (https://github.com/prestosql/presto/issues/1652) create images with HDP and KMS
+presto-product-tests/bin/run_on_docker.sh \
+    singlenode-kerberos-kms-hdfs-no-impersonation \
+    -g storage_formats \
+    || exit_code=1
+
+# We have docker images with KMS on CDH only. TODO (https://github.com/prestosql/presto/issues/1652) create images with HDP and KMS
+presto-product-tests/bin/run_on_docker.sh \
+    singlenode-kerberos-kms-hdfs-impersonation \
+    -g storage_formats \
+    || exit_code=1
+
+# Does not use hadoop
```

Comment:
actually there is hadoop up and running there, but might be not used for mysql tests (I am not sure about hadoop usage in mysql tests)

Reply:
I think it's not used (even though it's up and running).

if it is used, we will revisit this comment message. I still dont see a need to run these tests on multiple hadoop versions